src/cmdlinetest/test_afp_cli.t

```diff
@@ -48,6 +49,7 @@
    u?'--no-ask-pw': True, (re)
    u?'--show': False, (re)
    u?'--user': None, (re)
+   '--write': False,
```

Comment:
I.e. tests will fail on python3 w/o it

On 14 October 2015 10:46:41 CEST, Schlomo Schapiro <notifications@github.com> wrote:
>> @@ -48,6 +49,7 @@
>>     u?'--no-ask-pw': True, (re)
>>     u?'--show': False, (re)
>>     u?'--user': None, (re)
>> +   '--write': False,
>
>Why without the Unicode prefix? I am not sure if it matters, it just
>sticks out.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/ImmobilienScout24/afp-cli/pull/17/files#r41968461

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.